utils/postprocess/elasticity_postprocessing.py

```diff
@@ -165,4 +165,94 @@ def solutionFieldOnMeshToVtk(X, triangles, pointData, file_path):
     cell_types = np.ones(dol_triangles.shape[0])*5
     
     unstructuredGridToVTK(file_path, x, y, z, dol_triangles.flatten(), offset, 
-                          cell_types, pointData=pointData)
\ No newline at end of file
+                          cell_types, pointData=pointData)
+    
+def solutionFieldOnMeshToVtk3D(geom, 
+                               model, 
+                               save_folder_path = None,
+                               file_name = "3D_example",
+                               analytical_displacements=None, analytical_stresses=None, polar_transformation=None):
+    '''
+    Creates the vtu file based on geom and model for 3D elasticity
+    
+    Parameters
+    ----------
+    geom: object
+        The geometry object 
+    model: object
+        The trained model
+    save_folder_path: str
+        The path to the folder for storing the results
+    file_path: str:
+        The full file path to store the results
+    analytical_displacements: numpy array, 
+        Analytical displacement solutions. Order u_x, u_y, u_z
+    analytical_stresses: numpy array, 
+        Analytical stress solutions. Order sigma_xx, sigma_yy, sigma_zz, sigma_xy, sigma_yz, sigma_xz
+    polar_transformation: str
+        Coordinate system for polar transformation: spherical or cylindrical
+    '''
+    
+    X, offset, cell_types, elements = geom.get_mesh()
+
+    output = model.predict(X)
```

Comment:
Just as a comment / idea for later:

Maybe it makes sense to extract these lines (or at least the model evaluation) from this method. I'm just thinking about cases that we discussed yesterday, where we might want to compare the PINN prediction to that of an FE simulation which was solved on a different mesh. In that case we would need to evaluate the trained model not on the geometry used for training but on the nodes of the computational mesh coming from the FEM simulation. 

So maybe we could adapt the signature to pass an `output` tensor and then the `model` parameter would not be needed anymore (if I see it correctly).

Reply:
**EDIT**: Okay, thinking further about this, we would still have to somehow take care of the elements, so it's not that straightforward.. Let's maybe discuss this in person :)

Reply:
I would leave this for the time being as is. Because if there is a solution which is defined on another mesh or point cloud, such as FEM, this is another story. One has to generate own mesh, connectivity and and so on. But this can be done inside of the postprocessing method as well. Let's leave this as is for now.